lib/Repository.js

```diff
@@ -405,6 +405,20 @@ class Repository extends Requestable {
    }
 
    /**
+    * Get the README of a repository
+    * @see https://developer.github.com/v3/repos/contents/#get-the-readme
+    * @param {string} ref - the ref to check
+    * @param {boolean} raw - `true` if the results should be returned raw instead of GitHub's normalized format
+    * @param {Function} cb - will receive the fetched data
+    * @return {Promise} - the promise for the http request
+    */
+   getReadme(ref, raw, cb) {
+      return this._request('GET', `/repos/${this.__fullname}/readme`, {
```

Comment:
Thanks for matching the existing style (if only that was what I meant for it to be, stupid auto-formatters 😜), I'll clean both of them up. They'll both be:

```javascript
get*(ref, raw, cb) {
  return this._request('GET', `/repos/${this.__fullname}/readme`, {ref}, cb, raw);
}
```